hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -90,6 +90,11 @@
   // It is here so that both the client and deltastreamer use the same reference
   public static final String DELTASTREAMER_CHECKPOINT_KEY = "deltastreamer.checkpoint.key";
 
+  public static final ConfigProperty<String> DATABASE_NAME = ConfigProperty
```

Comment:
better to point to the definition of `HoodieTableConfig.DATABASE_HOME` directly, to avoid define repeatedly.

Reply:
Yes, just to keep consistent with other parameters before. If not, don't  need to change other parameters for the time being? Is it better to revise it uniformly?

Reply:
just changing the configs related to this pr is ok.